### ISSUE TITLE
Some confusion about the dataset

### ISSUE THREAD

**Issue by LCJHust** [nischnei/depth_completion#3]
Hi!@nischnei 👍
     I have some questions about the SparseConvNet.
 1. What is the purpose of Synthia dataset of Ros? Does the paper mean that we train the SparseConvNet with the Synthia dataset which is 5% sparsity, then use the model to generate the KITTI depth completion dataset on KITTI raw dataset?
 2. How should I construct the loss function?

I really look forward your reply.

**Comment by nischnei** [comment]
Hey @LCJHust!
We did different experiments. One was as you mentioned, that we trained on Synthia at e.g. 5% sparsity and tested on KITTI. However, the numbers reported on the KITTI page were achieved by training on the KITTI training dataset and then testing on the KITTI test dataset.

We currently just used L2 loss.

Be aware, that the paper just gives a comparison between sparse convolutions and standard convolutions - we used a very weak network with only a few layers and only limited depth in each layer. You could definitely use more tricks and more powerful networks. Have a look at some of the other methods that are published at the KITTI depth completion benchmark!

**Comment by LCJHust** [comment]
@nischnei 
      Thank you for your reply!

**Comment by CC9310** [comment]
@nischnei 
the max_pooling layer in sparse_conv where you set the stride==1,I think it is not right,and the code output is not right as paper said.
appreciate for your reply.

**Comment by nischnei** [comment]
@DiscoDiscoBoomBoom thank you for the feedback. There might have been an old version of the code. I did an extensive update, please check if everything is fine for you now!
